### PR TITLE
Addressing comments

### PR DESCRIPTION
**Pull request by aaronchongth**
Signed-off-by: Aaron Chong <aaronchongth@gmail.com>

Addresses comments in https://github.com/RobotLocomotion/drake-ros/pull/20

---
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake-ros/35)

### CODE REVIEW

File: drake_ros_tf2/test/test_name_conventions.cc [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+
+#include "../src/utilities/internal_name_conventions.h"
```
Comment (hidmic) [inline drake_ros_tf2/test/test_name_conventions.cc:17]: @aaronchongth consider using `target_include_directories` to add `src` or `src/utilities` as `BUILD_INTERFACE` include path.

File: drake_ros_tf2/src/utilities/internal_name_conventions.h [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#pragma once
+
+#include <sstream>
+#include <string>
+
+namespace drake_ros_tf2 {
+namespace utilities {
+
```
Comment (hidmic) [inline drake_ros_tf2/src/utilities/internal_name_conventions.h:21]: @aaronchongth nit: should this be in an `internal` namespace

**Comment by hidmic** [comment]
This looks awesome @aaronchongth ! I'll let @sloretz give the approval though.

File: drake_ros_tf2/test/test_name_conventions.cc (hunk 1 above)
Comment (aaronchongth) [reply drake_ros_tf2/test/test_name_conventions.cc]: Good call, that would make the includes much cleaner, thanks! https://github.com/RobotLocomotion/drake-ros/pull/35/commits/a9cf24a09ecb598e648496f411ec301610746ab8

File: drake_ros_tf2/src/utilities/internal_name_conventions.h (hunk 2 above)
Comment (aaronchongth) [reply drake_ros_tf2/src/utilities/internal_name_conventions.h:22]: Makes sense! Added it in https://github.com/RobotLocomotion/drake-ros/pull/35/commits/a9cf24a09ecb598e648496f411ec301610746ab8 as well

**Comment by aaronchongth** [comment]
Thanks for the review, @hidmic! Sounds good. Let me know what you think, @sloretz

**Review by sloretz — APPROVED** [review]

**Comment by hidmic** [comment]
I'll rebase merge onto #20 so we can move forward with that one.

**Pull request merged**